sdk/batch/batch/package.json

```diff
@@ -72,7 +74,7 @@
     "clean": "find src -type f -name \"*.js\" -delete",
     "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/batch.js.map'\" -o ./dist/batch.min.js ./dist/batch.js",
     "prepack": "npm install && npm run build",
-    "test": "ts-mocha -p tsconfig.test.json test/*.spec.ts --timeout 100000"
+    "test": "mocha -r esm --require ts-node/register test/*.spec.ts --timeout 1200000 --full-trace"
```

Comment:
@joheredi can we update the code generator to generate this testing script instead that can be run easily by the automatic configuration in vs code debugger?

Reply:
opened https://github.com/Azure/autorest.typescript/issues/766.